src/components/Card.js

```diff
@@ -48,12 +38,36 @@ export default class Card {
     });
   }
 
-  _handleDeleteButton() {
+  cardIsLiked() {
+    return this._isLiked;
+  }
+
+  handleDeleteButton = () => {
     this._cardElement.remove();
     this._cardElement = null;
-  }
+  };
 
-  _handleLikeButton() {
+  handleLikeButton = () => {
     this._cardLikeButton.classList.toggle("liked");
+  };
+
+  getView() {
+    this._getTemplate();
+
+    this._cardLikeButton =
+      this._cardElement.querySelector(".card__like-button");
+    this._cardDeleteButton = this._cardElement.querySelector(
+      ".card__delete-button"
+    );
+    this.cardTitleElement = this._cardElement.querySelector(".card__title");
+    this.cardImageElement = this._cardElement.querySelector(".card__image");
+
+    this.cardTitleElement.textContent = this._name;
+    this.cardImageElement.src = this._link;
+    this.cardImageElement.alt = this._name;
+
```

Comment:
Needs correction: don't forget to set the initial likes button state, use `renderLikes` method for that